dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.js

```diff
@@ -305,22 +305,45 @@ dojo.declare(
             }
         },
 
+        _getSiteFolderFieldDefaultHTML: function (){
+            const defaultSiteFolderField = {
+                "fieldContentlet": "system_field",
+                "fieldFieldType": "host or folder",
```

Comment:
shouldn't this fieldFieldType be "host_or_folder" ?  I mean blanks in a type ?

Reply:
Looks like this value comes from this const
`HOST_OR_FOLDER("host or folder")`
approving

Reply:
`Field.FieldType.HOST_OR_FOLDER.toString()` can only be accessed in JSP or Java files